Servers/UI/OJS.Servers.Ui/ClientApp/src/hooks/use-urls.tsx

```diff
@@ -47,26 +48,26 @@ const getParticipationsForProfileUrl = () => `${baseUrl}/Participations/GetForPr
 
 // contests
 const getIndexContestsUrl = () => `${baseUrl}/Contests/GetForHomeIndex`;
-const getAllContestsUrl = ({ filters }: IAllContestsUrlParams) => {
-    let statusParams = filters.filter((f) => f.type === FilterType.Status);
-    if (statusParams.length === 2) {
-        statusParams = [];
-    }
-
-    const statusParam = statusParams.map(({ name }) => `filter=${name}`)
-        .join('');
+const getAllContestsUrl = ({ filters, page }: IAllContestsUrlParams) => {
+    const queryParams = `${filters
+        .map(({ value, type }) => `${type.toLowerCase()}=${value}`)
+        .join('&')
+    }&page=${page}`;
 
     return (
-        isNil(statusParam)
+        isNil(queryParams)
```

Comment:
Is this check necessary? `queryParams` will always be atleast `&page=${page}`, right?